### PR TITLE
add WebSockets support

### LINKED ISSUE #105: ws / wss support
## Problem or use case
Is there one already or are you planning to add ws / wss support ?

### PR DESCRIPTION
**Pull request by gitseti**
**Motivation**
resolves #105 

**Changes**
- Add `-ws` option in `mqtt pub`, `mqtt sub` and `con` to use websockets transport protocol

### CODE REVIEW

File: src/main/java/com/hivemq/cli/commands/AbstractCommonFlags.java [hunk 1]
```diff
@@ -324,5 +329,16 @@ public void setCleanStart(final @Nullable Boolean cleanStart) {
         this.cleanStart = cleanStart;
     }
 
+    @Nullable
+    public MqttWebSocketConfig getWebSocketConfig() {
+        if (useWebsockets) {
+            return MqttWebSocketConfig.builder()
+                    .serverPath("/mqtt")
```
Comment (SgtSilvio) [inline src/main/java/com/hivemq/cli/commands/AbstractCommonFlags.java]: Although we are using this path on broker.hivemq.com, should this be really the default?

Comment (gitseti) [reply src/main/java/com/hivemq/cli/commands/AbstractCommonFlags.java]: Hmm, i thought this is the standard path for mqtt over websockets.

What do you think about adding a `-ws:path` option to specify the path and using `/mqtt` via default properties?

Comment (SgtSilvio) [reply src/main/java/com/hivemq/cli/commands/AbstractCommonFlags.java]: yes, this is good idea

Comment (SgtSilvio) [reply src/main/java/com/hivemq/cli/commands/AbstractCommonFlags.java]: We should probably also support passing a single url like
`ws://broker.hivemq.com:8000/mqtt` or `wss://broker.hivemq.com:8001`

**Pull request merged**